sanic_swagger/serializer.py

```diff
@@ -5,15 +5,19 @@
     Any,
     Collection,
     Dict,
-    GenericMeta,
     Iterable,
     List,
     Mapping,
     Sequence,
     Set,
     Union,
 )
-
+# Check if we are running on Python 3.7
+import sys
+if sys.version_info[:3] >= (3, 7, 0):
+    GenericMeta = type
+else:
+    from typing import GenericMeta
```

Comment:
You'll need to add ` # noqa` to the end of this file, so that the linter doesn't complain about this import being not at the top of the file.